prov/usnic/src/fi_ext_usnic.h

```diff
@@ -78,4 +78,15 @@ struct fi_usnic_ops_av {
 	int (*get_distance)(struct fid_av *av, void *addr, int *metric);
 };
 
+/*
+ * usNIC-specific provider completion status errnos
+ */
+enum {
+	FI_EXT_USNIC_COMPSTAT_SUCCESS,
+	FI_EXT_USNIC_COMPSTAT_ERROR_CRC,
+	FI_EXT_USNIC_COMPSTAT_ERROR_TRUNC,
```

Comment:
I thought there was an 'TRUNC' flag or error code, but I don't see it.  This too makes sense as a generic error code.